libminifi/CMakeLists.txt

```diff
@@ -48,6 +49,19 @@ file(GLOB SPD_SOURCES "../include/spdlog/*")
 add_library(spdlog STATIC ${SPD_SOURCES})
 add_library(minifi STATIC ${SOURCES})
 
+# Include Boost System
+find_package(Boost COMPONENTS system REQUIRED)
+target_link_libraries(minifi ${Boost_SYSTEM_LIBRARY})
+
+# Include libxml2
```

Comment:
@apiri I *think* this was something that was there already, not something required for civet. If you're sure it's not needed elsewhere, I can take it out and test to make sure civet doesn't need it.